src/main/java/seedu/address/logic/commands/AddEventCommand.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+
+
+public class AddEventCommand extends Command {
+    public static final String COMMAND_WORD = "add_event";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds an event to the event list. "
+            + "Parameters: "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_DATE + "DATE "
+            + PREFIX_ADDRESS + "ADDRESS\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_NAME + "NUS Career Fair 2023 "
+            + PREFIX_DATE + "2023-12-23 "
+            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25";
+
+    public static final String MESSAGE_SUCCESS = "New event added: %1$s";
+    public static final String MESSAGE_DUPLICATE_EVENT = "This event already exists in the event list";
+
+    private final Event toAdd;
+
+    /**
+     * Creates an AddEventCommand to add the specified {@code Event}
+     */
+    public AddEventCommand(Event event) {
+        requireNonNull(event);
+        toAdd = event;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        if (model.hasEvent(toAdd)) {
+            throw new CommandException(MESSAGE_DUPLICATE_EVENT);
+        }
+        model.addEvent(toAdd);
+        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
```

Comment:
Consider formatting the event display in Messages.java. So it can be displayed without the `seedu.address.event.event`.

Reply:
fixed.